code/index.html

```diff
@@ -1,22 +1,146 @@
   <head>
-    <meta charset="utf-8" />
-    <title>Project 2: The News Site</title>
+    <meta charset="utf-8"
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>Project 2: News-site Economics Today</title>
     <link rel="stylesheet" href="./style.css" />
-    <!-- dont forget the meta tag for responsiveness -->
   </head>
+  
   <body>
-    <header>
-      <!-- Some kind of logo and navbar -->
+    <!--***Header desktop***-->
+    <header class="header sticky-top">   
+       <nav class="menu">
+          <a href="https://www.technigo.io/">
+            <img class="logo" 
+            src="./logo.svg"
+            alt="Technigo-company-logo"/>  
+         </a>
+         <div class="site-name"><b>ECONOMICS TODAY</b></div>    
+         <ul class="menu-links"> 
+           <li class="link"><a href="#">ABOUT</a></li>
+           <li class="link"><a href="#">CONTACT</a></li>
+           <li class="link"><a href="#">READ MORE</a></li>
+           <li class="link"><a href="#">PRESS</a></li>
+         </ul>   
+      </nav>
+   </header>
+   <!--***Header tablet***-->
+   <header class="header-tablet sticky-top">   
+        <a class="item-link" href="#">ABOUT</a>
+        <a class="item-link" href="#">CONTACT</a>
+        <a class="item-logo" href="https://www.technigo.io/"><img class="logo-tablet" src="./logo.svg" alt="Technigo logo"/></a> 
+        <a class="item-link" href="#">READ MORE</a>
+        <a class="item-link" href="#">PRESS</a>        
     </header>
-
-    <section class="big-news">
-      <!-- The big news content -->
+     <!--***Big news section***-->
+    <section class="big-news" name="top">
+       <div class="big-news-container">
+          <a class="big-news-image" href="#">              
+           <img class="tesla-car" src="https://images.unsplash.com/photo-1576221162298-3d9f04e9f661?ixid=MXwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHw%3D&ixlib=rb-1.2.1&auto=format&fit=crop&w=1050&q=80" 
+           alt="image-tesla-car"/>
+         </a>
+       </div>
+       <div class="big-news-container">   
+         <div class="big-news-text"> 
+             <a href="#">
+              <h1>Tesla starts delivering Shanghai-made Model Y in China</h1>     
+              <p class="hide-on-mobile">&#127757 Tesla Inc said on Monday it has started delivering its Shanghai-made Model Y sports utility vehicles to customers in China.</p>
+              <p class="hide-on-desktop" style="color:royalblue"><b>Read more here</b></p>
+             </a>
+          </div>  
+        </div> 
+    </section>      
+    <!--***News-card section***-->
+    <section class="other-news">
+      <div class="news">
+        <div class="card">
+          <a href="#">
+            <img 
+            src="https://images.unsplash.com/photo-1594810205183-18a8b0ce6c13?ixlib=rb-1.2.1&ixid=MXwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHw%3D&auto=format&fit=crop&w=1050&q=80" 
+            alt="image eu-flags"
+            />            
```

Comment:
Real nice that you're giving the attributes new lines! Just remember to indent as well:
```
<img 
  src="https://images.unsplash.com/photo-1594810205183-18a8b0ce6c13?ixlib=rb-1.2.1&ixid=MXwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHw%3D&auto=format&fit=crop&w=1050&q=80" 
  alt="image eu-flags"
/> 
```